osc-staging.py

```diff
@@ -193,16 +188,12 @@ def do_staging(self, subcmd, opts, *args):
         elif cmd == 'accept':
             # Is it safe to accept? Meaning: /totest contains what it should and is not dirty
             version_totest = api.get_binary_version(api.project, "openSUSE-release.rpm", repository="totest", arch="x86_64")
-            skip_totest = False
-            if not version_totest:
-                # SLE don't have totest repository and openSUSE-release.rpm
-                skip_totest = api.item_exists(api.project, "release-notes-sles")
 
-            if skip_totest or opts.force:
+            if not version_totest or opts.force:
```

Comment:
will it being better for Leap, looks Leap have to use force anyway here, since it has no totest manager involved